dbld/packages.manifest

```diff
@@ -96,8 +96,9 @@ python3-pip                     [debian, ubuntu]
 
 # python 3.10 has issues on Debian Testing
 # Nevertheless we should only downgrade for Light style-check tests, which uses pre-commit
-python3.9                   [devshell]
-python3.9-venv              [devshell]
+python3.11                   [devshell]
+python3.11-dev               [devshell]
```

Comment:
IMHO the `-dev` package is not needed. This python installation for `devshell` is added so the Light style-check (using pre-commit) works, it's not for syslog-ng's python dependency.
python3 is installed based on `packaging/debian/control` and `dbld/build.manifest` files.

Installation of this specific python-version was added in e61d4cf2caa49d108f28bfa884763d1bbb4d990b when we found a pre-commit issue.

Reply:
Thanks for the review. Unfortunately `python3.11-dev` package will be required (if possible I would skip it also), due to `psutil` package will not be installed (built) without it during `pip install`. Yes, I know previously it was not required, but in `python3.11` it needed.

Reply:
(psutil is currently compiled from source with the current state of pypi and debian-testing)